### ISSUE TITLE
 How can l get the order of human key points  and their associated feature vectors ?

### ISSUE THREAD

**Issue by pinkfloyd06** [ZheC/Realtime_Multi-Person_Pose_Estimation#192]
Hello @ZheC,

Let me first thank you for your clean code and paper. 

I have a question related to the order of key points.

  1) Since we have 18 human key points, l would like to know which key point belong to tendon, knee, arm, leg ... ?

  2) Is there a way to get a feature vector describing each human key point ?

  3)  What does out6= torch.cat([out5_1,out5_2,out1],1) represent ?
    It is of dim(185,x,y). What is 185 ?

Thank you a lot for you answer
